strawberryfields/backends/bosonicbackend/backend.py

```diff
@@ -553,7 +553,7 @@ def mbsqueeze(self, mode, r, phi, r_anc, eta_anc, avg):
             return ancilla_val
 
     def beamsplitter(self, theta, phi, mode1, mode2):
-        self.circuit.beamsplitter(-theta, -phi, mode1, mode2)
+        self.circuit.beamsplitter(theta, phi, mode1, mode2)
```

Comment:
This was changed because internally we use the conventions of the walrus which are consistent with the frontend of SF, right?

Reply:
Exactly. This allows the beamsplitter tests to pass as a consequence.